docs/production/accounts.md

```diff
@@ -4,64 +4,66 @@ List of accounts used by Dark as part of running Dark
+- `ops` account on [darklang.com](https://darklang.com)
+  we maintain an `ops` account shared by Dark employees.
+  - `ops-adduser`
+  - `ops-circleci`
+  - `ops-corp-site`
+  - `ops-login`
+  - `ops-presense`
+- [Slack (dark community)](https://darklang.com/slack-invite)
 
 # Deploying
 
-- circleci.com
-- launchdarkly.com
+- [CircleCI](https://app.circleci.com/pipelines/github/darklang) for continuous integration
+- [LaunchDarkly](https://launchdarkly.com) for feature flags
 
 # Hosting
 
-- cloud.google.com
+- [Google Cloud](http://cloud.google.com)
   - GKE
   - Cloud SQL
   - Load Balancers
-  - PubSub
-  - CDN
+  - CDN for static assets (both ours and users')
   - DNS
   - Google Domains
+  - PubSub for queues/crons
   - support
-- auth0
-- pusher.com
-- algolia (docs)
-- cloudflare.com
-  - not really used, some DNS
-- positivessl/sectigo
-  - certs for darksa.com
-- let's encrypt
-  - cert manager
-  - customdomains
-- brex
-  - how we pay for everything
-- name.com
-- ghost.org
-  - blog.darklang.com
-- todesktop.com
-  - desktop client
-- ngrok.com
+- [auth0](https://manage.auth0.com/dashboard) user account management
+- [pusher.com](https://dashboard.pusher.com/apps) supports "real-time" pushes of information to the Editor
+- [algolia](https://www.algolia.com) search engine used to support [Darklang docs](https://docs.darklang.com)
+- [CloudFlare](https://cloudflare.com) slight usage for some DNS
+- [PositiveSSL](https://www.positivessl.com) manages certificates for darksa.com
+- [let's encrypt](https://letsencrypt.org) certificate management and custom domains
+  see [custom-domains](./custom-domains.md)
+- [brex](https://www.brex.com) how we pay for everything
+- [name.com](http://name.com) TODO what do we use this for?
+- [Ghost.org](https://ghost.org) powers [the blog](https://blog.darklang.com)
+- [ToDesktop](https://todesktop.com) manages the desktop client of Dark
+- [ngrok](https://ngrok.com) useful for exposing localhost ports externally
 
 # Monitoring
 
-- honeycomb.io
-- rollbar.com
-- betteruptime.com
-- pagerduty.com
+- [Honeycomb](https://honeycomb.io) for monitoring, production analysis, etc.
+  see [honeycomb.md](./honeycomb.md)
+- [Rollbar](https://rollbar.com) for recording, exploring, and resolving bugs
+- [PagerDuty](https://pagerduty.com) for alerting when things are broken
   - triggers from rollbar, pingdom and honeycomb
-- pingdom.com
-- sslmate.com
-- drlinkcheck.com
+- [Better UpTime](https://betteruptime.com) for alerting when things are broken (newer)
+- [Pingdom](https://pingdom.com) TODO: where do we use this?
+- [sslmate](https://sslmate.com) TODO: where do we use this?
+- [Dr. Link Check](https://drlinkcheck.com) TODO: where do we use this?
 
-# User comms, analytics
+# User communications, analytics
 
-- mailchimp
-- mandrill (automated emails)
-- heap analytics
-  - not really used
-- full story
-  - not really used
+- [Mailchimp](https://mailchimp.com) TODO: where do we use this?
```

Comment:
open question

Reply:
We add users to it when they sign up in ops-adduser canvas, and then we use it to email users when there are product updates.